egs/fisher_swbd/s5/local/chain/run_tdnn_lstm_1a_svd.sh

```diff
@@ -0,0 +1,378 @@
+#!/bin/bash
+#
+# Copyright 2018  Nagendra Kumar Goel,
+#            Saikiran Valluri, Govivace.Inc -  Apache 2.0
+
+# The script is organized as below.
+# First we train the baseline LSTMP-TDNN config chain model for few epochs on the multi-english data,
+# Then, we perform SVD based refactoring of all the Affine components in this baseline final.mdl,
+# in order to reduce the overall model parameters size,
+# as determined by the bottleneck dim value or Energy and Shrinkage threshold values.
+# Then, we finetune the weight parameters of the refactored model using entire multien data for single epoch.
+
+# Command used for comparing  WER decoding on testsets :
+#  ./local/chain/compare_wer_general.sh --looped tdnn_lstm_1a_sp tdnn_lstm_1a_svd_sp
+
+
+# System                tdnn_lstm_1a_sp
+# WER on eval2000(tg)        12.3
+#           [looped:]        12.2
+# WER on eval2000(fg)        12.1
+#           [looped:]        12.1
+# WER on eval2000(fg)
+#  [SVD retrained + looped]  12.1
+# WER on rt03(tg)            11.6
+#           [looped:]        11.6
+# WER on rt03(tg)
+#  [SVD retrained]           12
+# WER on rt03(fg)            11.3
+#           [looped:]        11.3
+# Final train prob         -0.074
+# Final valid prob         -0.084
+# Final train prob (xent)        -0.882
+# Final valid prob (xent)       -0.9393
+
+# ./show_chain_wer.sh tdnn_lstm_1b_sp
```

Comment:
`tdnn_lstm_1b_sp`? Or maybe you should show for `tdnn_lstm_1a_svd_sp`?

Reply:
Actually the WERs for all testsets for the SVD retrained model and pre-SVD were reported 2 years ago by Navneeth, who left the company 1 year ago. I am continuing the PR ownership. So, now since Dan reported rt03 dataset WER missing for SVD , I just ran the rt03 decoding for SVD and reported it in the comments of the shell script.

Reply:
I have updated the WERs for the rt03 test for SVD retrained model at line 54 in local/chain/run_tdnn_lstm_1a_svd.sh.